src/router/kromo/dd-wrt/elegant/fresh.css

```diff
@@ -79,23 +80,24 @@ input {
   font-size: 12px;
   line-height: 1.42857;
   color: #555;
-  background-color: #FFF;
+  background-color: #fff;
   background-image: none;
-  border: 1px solid #CCC;
+  border: 1px solid #ccc;
   border-radius: 4px;
-  box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.075) inset;
-  transition: border-color 0.15s ease-in-out 0s, box-shadow 0.15s ease-in-out 0s;
```

Comment:
`0px` is only `0` unnecessary crud and also `0.075` you dont need it `.075` is enough, now if you remove the dot it matters.

none of these changes ADD bits to file size.